pkg/rbd/nodeserver.go

```diff
@@ -102,6 +103,21 @@ func (ns *NodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublis
 	return &csi.NodePublishVolumeResponse{}, nil
 }
 
+func (ns *NodeServer) NodeExpandVolume(ctx context.Context, req *csi.NodeExpandVolumeRequest) (*csi.NodeExpandVolumeResponse, error) {
+
+	diskMounter := &mount.SafeFormatAndMount{Interface: ns.mounter, Exec: mount.NewOsExec()}
+
+	//TODO: rework on the path
+	resizer := resizefs.NewResizeFs(diskMounter)
+	ok., err :=  resizer.Resize(req.getVolumeId(), req.getVolumePath())
```

Comment:
Nit: only if the PVC is in FS mode (i.e. don't attempt to expand a non-existent FS on a block PVC)?

Reply:
Yep, Have to consider both file/block modes. I have put this PR in WIP mainly to expand on this function. Will be on it soon.

Reply:
Done